addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4658,6 +4658,7 @@ registry.SnippetMove = SnippetOptionWidget.extend({
  * Allows for media to be replaced.
  */
 registry.ReplaceMedia = SnippetOptionWidget.extend({
+    xmlDependencies: ['/web_editor/static/src/xml/link_tool.xml'],
```

Comment:
Elsewhere in odoo the name is "link_tools" instead of "link_tool". Do we want a different name for this one ?
Or maybe "image_link_tools" ?
(If changed, also update template ids accordingly)